test/e2e/upgrade_test.go

```diff
@@ -190,19 +193,26 @@ var _ = Describe("BMO Upgrade", func() {
 			}
 		} else {
 			By("Creating a separate cluster for upgrade tests")
+			upgradeClusterName = fmt.Sprintf("bmo-e2e-upgrade-%d", GinkgoParallelProcess())
```

Comment:
So the clusters will all be named the same when running with one worker? :thinking: 
This does not feel right. I would prefer to name it something based on the version it is going to test.

Reply:
Hmm. May I ask why? I don't think the cluster name is that important, we only named it "bmo-e2e-upgrade" before. The cluster name is not logged anywhere else, and the clusters themselves will be cleaned afterwards. IMO the only reason it needs to be unique is so that the tests can be ran in parallel.

Making the name based on version requires somewhat hacky change. And the names will get messy real fast once we introduce ironic upgrade tests.

Reply:
Ok, it was not as bad as I thought. I remembered how the cluster name is the folder where the logs are stored in CAPI/CAPM3 tests. It is fine like this :slightly_smiling_face: